.env

```diff
@@ -0,0 +1,5 @@
+REST_API_URL="http://localhost:9966/petclinic/api/"
+SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/petclinic
+SPRING_DATASOURCE_USERNAME=postgres
+SPRING_DATASOURCE_PASSWORD=petclinic
```

Comment:
We should remove these variables for Spring. They are only used on the backend. No need for them here.